pynamodb/models.py

```diff
@@ -886,41 +879,24 @@ def _get_schema(cls) -> Dict[str, Any]:
                     'key_type': RANGE,
                     'attribute_name': attr_cls.attr_name
                 })
-        return schema
-
-    @classmethod
-    def _get_indexes(cls):
-        """
-        Returns a list of the secondary indexes
-        """
-        index_data: Dict[str, List[Any]] = {
-            'global_secondary_indexes': [],
-            'local_secondary_indexes': [],
-            'attribute_definitions': []
-        }
-        for name, index in cls._indexes.items():
-            schema = index._get_schema()
-            idx = {
-                'index_name': index.Meta.index_name,
-                'key_schema': schema['key_schema'],
-                'projection': {
-                    PROJECTION_TYPE: index.Meta.projection.projection_type,
-                },
-            }
+        for index in cls._indexes.values():
+            index_schema = index._get_schema()
             if isinstance(index, GlobalSecondaryIndex):
-                if getattr(cls.Meta, 'billing_mode', None) != PAY_PER_REQUEST_BILLING_MODE:
-                    idx['provisioned_throughput'] = {
-                        READ_CAPACITY_UNITS: index.Meta.read_capacity_units,
-                        WRITE_CAPACITY_UNITS: index.Meta.write_capacity_units
-                    }
-            index_data['attribute_definitions'].extend(schema['attribute_definitions'])
-            if index.Meta.projection.non_key_attributes:
-                idx['projection'][NON_KEY_ATTRIBUTES] = index.Meta.projection.non_key_attributes
-            if isinstance(index, GlobalSecondaryIndex):
-                index_data['global_secondary_indexes'].append(idx)
+                schema['global_secondary_indexes'].append(index_schema)
             else:
-                index_data['local_secondary_indexes'].append(idx)
-        return index_data
+                schema['local_secondary_indexes'].append(index_schema)
+        attr_names = {key_schema[ATTR_NAME]
+                      for index_schema in chain(schema['global_secondary_indexes'], schema['local_secondary_indexes'])
```

Comment:
```suggestion
                      for index_schema in(*schema['global_secondary_indexes'], *schema['local_secondary_indexes'])
```

Reply:
is the second one more "pythonic"?

Reply:
🤷  it's a Python 3.5 thing so we didn't have it before (though `itertools.chain` still has one thing on this syntax, in that you can do `chain(*list_of_lists)`)